integration_test/ginkgo_integration_test.go

```diff
@@ -0,0 +1,61 @@
+package integrationtest
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/node-disk-manager/integration_test/minikube_adm"
+	"github.com/openebs/node-disk-manager/integration_test/ndm_util"
+)
+
+func TestIntegrationNDM(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Integration Test Suit")
+}
+
+var _ = BeforeSuite(func() {
+	// It starts minikube if it is not Running
+	minikubeadm.Setup()
+
+	// It waits till namespace is ready
+	ndmutil.WaitTillDefaultNSisReady()
+
+	// It applies our CRDs
+	ndmutil.ApplyCRDs()
+
+	// It prepares configuration and Applies the same
+	ndmutil.PrepareAndApplyYAML()
+
+	// It waits till node-disk-manager is ready
+	ndmutil.WaitTillNDMisUp()
+})
+
+var _ = AfterSuite(func() {
+	// It Delete minikube if it is running
+	// It removes residue containers
+	// It removes remaining residue files
+	ndmutil.Clean()
+})
+
+var _ = Describe("Integration Test", func() {
+	// Now as BeforeSuit has run, We shall have a healthy node-disk-manager daemonset
+	When("We check the log", func() {
+		It("has `Started the controller` in the log", func() {
+			validated, err := ndmutil.GetNDMLogAndValidate()
+
+			Expect(err).NotTo(HaveOccurred())
+			Expect(validated).To(BeTrue())
+		})
+	})
+
+	When("We check Disk", func() {
```

Comment:
These 4 lines are not human readable. Can we rephrase it such that it turns out to be a readable paragraph?

Reply:
These are mostly the blank lines to complete the functions scope. This is where `.feature` files make more sense. For 4th line should I change it to `When("We check the disk", func() {`? Or do you have any other suggestions?